zephyr/src/rw61x/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ if(CONFIG_NXP_MONOLITHIC_WIFI OR CONFIG_NXP_MONOLITHIC_NBU)
 
 		set(signed_binary_blob_name rw61x_sb_wifi_a2.bin)
 
-		list(APPEND signed_binary_blobs_list ${hal_blobs_dir}/${signed_binary_blob_name})
+		list(APPEND signed_binary_blobs_list ${hal_blobs_dir}/rw61x/${signed_binary_blob_name})
 
```

Comment:
`hal_blobs_dir` already has rw61x as seen below. 
`set (hal_blobs_dir ${ZEPHYR_HAL_NXP_MODULE_DIR}/zephyr/blobs/rw61x)`
 
There is no need to change this file. Why do you think we need this change?